### PR TITLE
feat: alert user when engine container actions fail

### LINKED ISSUE #1809: Podman machine extension does not error out when start fails
When an error occurs during start of the podman machine, like missing disk file (inconsistency between WSL and podman machine), out of disk space, out of memory, or other WSL related issues, the user will not see this.

image

the button remains in a start state, so this can be pressed multiple times and no feedback is given.

### PR DESCRIPTION
**Pull request by lstocchi**
### What does this PR do?

This PR shows a short message next to the status in the resources page and add the full error text in the task view.

To keep it simple i didn't create any new component to show the full error, we can discuss what and where to create it here.
 
### Screenshot/screencast of this PR

error-message-start

### What issues does this PR fix or reference?

this is part of #1337 
this fixes https://github.com/containers/podman-desktop/issues/1809

### How to test this PR?

1.  Just start a podman machine that cannot be started. I just brutally added the same error text as in the issue to see how a long message would look like in the task view.

### CODE REVIEW

**Review by benoitf — APPROVED** [review]
image

**Review by benoitf — COMMENTED** [review]
looks like in the PR there are a lot of variables being renamed so it's harder to find what is like a new thing and just a renaming. Would have been better to split in two PRs

**Comment by lstocchi** [comment]
> looks like in the PR there are a lot of variables being renamed so it's harder to find what is like a new thing and just a renaming. Would have been better to split in two PRs

Ok, i'll create a new one with the renaming 👍

**Comment by benoitf** [comment]
let's keep it like this, for the next time

File: packages/renderer/src/lib/preferences/PreferencesResourcesRendering.svelte [hunk 1]
```diff
@@ -113,42 +118,103 @@ $: providerContainerConfiguration = tmpProviderContainerConfiguration
...
+        );
+      }
+      await window.startProviderConnectionLifecycle(
+        provider.internalId,
+        containerConnectionInfo,
+        loggerHandlerKey,
+        eventCollect,
+      );
+    }
+  } catch (e) {}
```
Comment (benoitf) [inline packages/renderer/src/lib/preferences/PreferencesResourcesRendering.svelte]: probably log in the console ?

File: packages/renderer/src/lib/preferences/PreferencesResourcesRendering.svelte [hunk 2]
```diff
@@ -113,42 +118,103 @@ $: providerContainerConfiguration = tmpProviderContainerConfiguration
...
+        );
+      }
+      await window.stopProviderConnectionLifecycle(
+        provider.internalId,
+        containerConnectionInfo,
+        loggerHandlerKey,
+        eventCollect,
+      );
+    }
+  } catch (e) {}
```
Comment (benoitf) [inline packages/renderer/src/lib/preferences/PreferencesResourcesRendering.svelte]: `console.error(e)` ?

File: packages/renderer/src/lib/preferences/PreferencesResourcesRendering.svelte [hunk 3]
```diff
@@ -113,42 +118,103 @@ $: providerContainerConfiguration = tmpProviderContainerConfiguration
...
+        getLoggerHandler(provider, containerConnectionInfo),
+      );
+      await window.deleteProviderConnectionLifecycle(
+        provider.internalId,
+        containerConnectionInfo,
+        loggerHandlerKey,
+        eventCollect,
+      );
+    }
+  } catch (e) {}
```
Comment (benoitf) [inline packages/renderer/src/lib/preferences/PreferencesResourcesRendering.svelte]: console.error(e)

**Comment by benoitf** [comment]
in case of errors (starting a machine while there is another one active) I got a UnhandledPromiseRejectionWarning

image

on main side it's important to not have those

**Review by benoitf — APPROVED** [review]
tried with all the fixes and now it works as expected 👍

**Review by benoitf — COMMENTED** [review]
~~wrong PR comment~~

**Pull request merged**